crates/web-sys/webidls/enabled/WebGLRenderingContext.webidl

```diff
@@ -955,6 +955,19 @@ interface EXT_texture_filter_anisotropic
     const GLenum MAX_TEXTURE_MAX_ANISOTROPY_EXT = 0x84FF;
 };
 
+[NoInterfaceObject]
+interface EXT_texture_norm16
+{
+      const GLenum R16_EXT          = 0x822A;
+      const GLenum RG16_EXT         = 0x822C;
+      const GLenum RGB16_EXT        = 0x8054;
+      const GLenum RGBA16_EXT       = 0x805B;
+      const GLenum R16_SNORM_EXT    = 0x8F98;
+      const GLenum RG16_SNORM_EXT   = 0x8F99;
+      const GLenum RGB16_SNORM_EXT  = 0x8F9A;
+      const GLenum RGBA16_SNORM_EXT = 0x8F9B;
+};
+
```

Comment:
This should probably go into `WebGL2RenderingContext.webidl` instead, since it's a WebGL 2-only extension.

Apart from that, looks good to me.

Reply:
You are absolutely right. Thanks for catching this!